packages/contracts/src/libraries/helpers/Validation.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.8.25;
+
+import "../../interfaces/ICrossL2Inbox.sol";
+import {Predeploys} from "../Predeploys.sol";
+import {ICrossL2Prover} from "../../interfaces/ICrossL2Prover.sol";
+
+contract Validation {
+    ICrossL2Prover private crossL2Prover;
```

Comment:
also where is the prover being set

Reply:
Having a look at it

Reply:
yea @AtharvSan add a constructor to set crossL2Prover